framework/web/Session.php

```diff
@@ -183,6 +183,8 @@ public function destroy()
         if ($this->getIsActive()) {
             @session_unset();
             @session_destroy();
+            @session_start();
```

Comment:
This is obviously incorrect: `session_start()` will start new session, thus there is no way to close session anymore.